source/operations/checklists/hardware.rst

```diff
@@ -202,16 +208,85 @@ Storage
 
 .. cond:: not k8s
 
+   Recommended Storage Mediums
+   +++++++++++++++++++++++++++
+
    MinIO recommends using flash-based storage (NVMe or SSD) for all workload types and scales.
    Workloads that require high performance should prefer NVMe over SSD.
 
    MinIO deployments using HDD-based storage are best suited as cold-tier targets for :ref:`Object Transition ("Tiering") <minio-lifecycle-management-tiering>` of aged data.
    HDD storage typically does not provide the necessary performance to meet the expectations of modern workloads, and any cost efficiencies at scale are offset by the performance constraints of the medium. 
 
-   Format Drives as XFS
-   ++++++++++++++++++++
+   Use Direct-Attached "Local" Storage (DAS)
+   +++++++++++++++++++++++++++++++++++++++++
+
+   DAS-storage, such as locally-attached JBOD (Just a Bunch of Disk) arrays, provide significant performance and consistency advantages over networked (NAS, SAN, NFS) storage.
+
+   .. dropdown:: Network File System Volumes Break Consistency Guarantees
+      :class-title: note
+
+      MinIO's strict **read-after-write** and **list-after-write** consistency
+      model requires local drive filesystems.
+
+      MinIO cannot provide consistency guarantees if the underlying storage
+      volumes are NFS or a similar network-attached storage volume. 
+
+      For deployments that *require* using network-attached storage, use
```

Comment:
I don't like the idea of saying to use NFS at all, and v4 only mitigates cache loss in most scenarios, not all. Would just say dont or not recommended and leave it at that .